### PR TITLE
Add Avalanche network namespace (CAIP-2)

### PR DESCRIPTION
**Pull request by gergelylovas**
This PR includes a CAIP-2 specification for identifying blockchains in the Avalanche ecosystem

### CODE REVIEW

File: avalanche/caip2.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# blockchainID: 2JVSBoinj9C2J33VntvzYtVJNZdN2NKiwwKjcumHUWEb5DbBrm
+avax:8AJTpRj3SAqv1e80Mtl9em08LhvKEbkl
+
+```
+
+## References
+
+- [Fuji Testnet][] - Fuji Testnet in Avalanche official documentation
+- [Info API][] - Info API reference in Avalanche official documentation
+- [txID][] - Create Chain TX reference in Avalanche official documentation
```
Comment (bumblefudge) [inline avalanche/caip2.md:131]:
```suggestion
- [txID][] - Create Chain TX reference in Avalanche official documentation
- [Multichain Architecture][] - Overview of P-Chain, X-Chain, and C-Chain system

[Multichain Architecture]: https://docs.avax.network/learn/avalanche/avalanche-platform
```

File: avalanche/caip2.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+*For context, see the [CAIP-2][] specification.*
+
+## Rationale
+
+In CAIP-2 a general blockchain identification scheme is defined. This is the
+implementation of CAIP-2 for the Avalanche Ecosystem. 
+Blockchains in the "avax" namespace are validated by their `blockchainID`. 
+The `blockchainID` in Avalanche refers to the [txID][] that created the 
+blockchain on the Avalanche P-Chain.
```
Comment (bumblefudge) [inline avalanche/caip2.md:21]:
```suggestion
blockchain on the Avalanche [P-Chain][Multichain Architecture].
```

(Makes more sense after accepting the suggestion below in `## References` )

**Review by bumblefudge — APPROVED** [review]
LGTM, other than my added normref!

**Review by obstropolos — APPROVED** [review]

**Comment by gergelylovas** [comment]
@bumblefudge @obstropolos Thank you for the reviews! Can you please help with merging the PR?

**Comment by bumblefudge** [comment]
My mistake, merging now!

**Pull request merged**